tensorrt_llm/llmapi/llm_args.py

```diff
@@ -678,6 +682,32 @@ def _to_pybind(self):
             host_cache_size=self.host_cache_size,
             lora_prefetch_dir=self.lora_prefetch_dir)
 
+    @staticmethod
+    def create_from_pybind(
+            peft_cache_config: _PeftCacheConfig) -> "PeftCacheConfig":
+        # Some of the properties are optional in CPP but in python they have a default value and aren't optional,
+        # so copy their value only if they have a value in the CPP instance.
+        extra_kwargs = {}
+        if peft_cache_config.device_cache_percent is not None:
+            extra_kwargs[
+                "device_cache_percent"] = peft_cache_config.device_cache_percent
+        if peft_cache_config.host_cache_size is not None:
+            extra_kwargs["host_cache_size"] = peft_cache_config.host_cache_size
+
+        return PeftCacheConfig(
```

Comment:
The python PeftCacheConfig should be a superset of knobs of the pybind one, so maybe we could programmatically iterate its attributes rather than manually expanding them in case there may be some new fields added over time.